spec.md

```diff
@@ -843,6 +848,10 @@ message CreateVolumeRequest {
   // VOLUME_ACCESSIBILITY_CONSTRAINTS plugin capability, the SP MAY
   // choose where the provisioned volume is accessible from.
   TopologyRequirement accessibility_requirements = 7;
+
+  // This field is OPTIONAL. This allows the CO to specify the
+  // volume performance class parameters to apply. 
```

Comment:
I agree on not using K8s-specific terms, but we need specific and clear verbiage in the CreateVolume text that explains what this new field is for, and how it should be used, BOTH for the CO and the SP.

I am particularly worried about the potential for overlap between keys in the parameters and the mutable_parameters list. For example:
1) What should happen if a CO specifies a key for a mutable parameter in the existing parameters field?
2) What should happen if a CO specifies a key for an immutable parameter in the mutable parameters field?
3) What should happen if a CO specifies a key in both maps?
4) What should happen if there are different values associated with the two keys?
5) Is it valid for an SP to support the same key having different meanings in the different maps?

What I'm getting at is: how is having separate maps for existing parameters and mutable parameters functionally different from putting them in the same map? To be clear: I think they should be separate, because I think it gives us a better shot at detecting and reporting user errors, but we should think through what those errors are likely to look like, and how they will be caught and reported, and what kind of guidance will minimize the likelihood of problems.